openslides/core/config_variables.py

```diff
@@ -108,6 +108,22 @@ def get_config_variables():
     # Projector
 
     yield ConfigVariable(
+        name='projector_language',
+        default_value='browser',
+        input_type='choice',
+        label='Projector language',
+        choices=(
+            {'value': 'en', 'display_name': 'English'},
+            {'value': 'de', 'display_name': 'Deutsch'},
+            {'value': 'fr', 'display_name': 'Français'},
+            {'value': 'es', 'display_name': 'Español'},
+            {'value': 'pt', 'display_name': 'Português'},
+            {'value': 'cs', 'display_name': 'Čeština'},
+            {'value': 'browser', 'display_name': 'Current browser language'}),
```

Comment:
Add this browser string to site.js. (The language can be used in native language)